### PR TITLE
fix for #2072 - The "random album" plugin thinks it can function in the playlist browser

### PR DESCRIPTION
**Pull request by draxil**
Fix for #2072 - The "ugly" fix mentioned in that issue that should stop the random album plugin causing annoying effects within the playlists browser until the architecture is ready to enact one of the other fixes mentioned in that issue.

### CODE REVIEW

File: quodlibet/quodlibet/ext/events/randomalbum.py [hunk 1]
```diff
@@ -233,3 +233,10 @@ def unpause(self):
             app.player.next()
         except AttributeError:
             app.player.paused = True
+
+    def disabled_for_browser(self, browser):
+        if not browser.can_filter_albums():
```
Comment (declension) [inline quodlibet/quodlibet/ext/events/randomalbum.py]: Surely `return browser == 'Playlists' or not browser.can_filter_albums()` is neater?

Comment (draxil) [inline quodlibet/quodlibet/ext/events/randomalbum.py]: Oh actually the fact that you said "return" should have informed me as to what you meant.. I'll make that change.

**Comment by draxil** [comment]
Anything else I can do to help get this accepted or is it a case of waiting to catch @lazka 's eye?

**Comment by declension** [comment]
Sorry, been a bit busy. LGTM.

**Comment by declension** [comment]
Thanks

**Comment by draxil** [comment]
Thanks!

**Pull request merged**